tbtc-ts/src/sweep.ts

```diff
@@ -371,3 +374,29 @@ async function prepareInputSignData(
     previousOutputValue: previousOutput.value,
   }
 }
+
+/**
+ * Prepares the proof of a sweep transaction and submits it to the Bridge
+ * on-chain contract.
+ * @param transactionHash - Hash of the transaction being proven.
+ * @param mainUtxo - Recent main UTXO of the wallet as currently known on-chain.
+ * @param bridge - Interface to the Bridge on-chain contract.
+ * @param bitcoinClient - Bitcoin client used to interact with the network.
+ * @returns Empty promise.
+ */
+export async function proveSweep(
+  transactionHash: string,
+  mainUtxo: UnspentTransactionOutput,
+  bridge: Bridge,
+  bitcoinClient: BitcoinClient
+): Promise<void> {
+  const confirmations = await bridge.txProofDifficultyFactor()
+  const proof = await createTransactionProof(
```

Comment:
`proof` contains `Transaction`. Worth dropping a TODO this can be used instead of calling `bitcoinClient.getRawTransaction` once we have a `Transaction -> RawTransaction` converter implemented.

Reply:
Done.